content/engineering-practices-why-metrics.md

```diff
@@ -0,0 +1,41 @@
+At WorkSafe BC, I am thrilled to see Metrics as a key factor in everything, every initiative and at every level within our Enterprise DevOps division.  As practitioner of SAFe (Scalable Agile Framework for the Enterprise), we have many teams working across various initiatives serving all aspects of the business and we use metrics to frame business benefit and track realization.
+
+# Metrics are the Key to Success
+Let me list the Metrics I see in play, and why they are crucial to our collective success.
+1. **Performance Evaluation:** Metrics provide quantitative measures to evaluate the performance of software development processes, teams, and individual developers. They help identify areas of improvement and track progress over time.
+
+2. **Quality Assurance:** Metrics help assess the quality of software products by measuring factors such as code complexity, code coverage, defect density, and adherence to coding standards. This information enables teams to identify and address quality issues early in the development lifecycle.
+
+3. **Resource Management:** Metrics help in resource allocation by providing insights into project timelines, resource utilization, and budget management. They assist in identifying bottlenecks, optimizing workflows, and making informed decisions about resource allocation.
+
+4. **Risk Management:** Metrics help identify and mitigate risks associated with software development projects. By tracking metrics related to project scope, schedule adherence, and budget, teams can proactively manage risks and avoid potential project failures.
+
+5. **Process Improvement:** Metrics facilitate continuous process improvement by providing objective data to identify inefficiencies, optimize workflows, and implement best practices. They enable teams to measure the impact of process changes and make data-driven decisions to improve overall productivity and quality.
+
+6. **Communication and Transparency:** Metrics provide a common language for communication and collaboration among stakeholders, including developers, managers, customers, and other project stakeholders. They help foster transparency by providing visibility into project progress, performance, and quality.
+
+
+# Conclusion
+
+Overall, metrics play a vital role in software engineering by providing objective data to evaluate performance, ensure quality, optimize resources, manage risks, drive process improvement, and facilitate communication and transparency among project stakeholders.
+> ![Organization driven by metric](/images/engineering-practices-why-metrics-3.png)
```

Comment:
@MerlinDarkstar  please add an empty line before and after the > callout, otherwise html formatting goes weird.